Dockerfile

```diff
@@ -1,20 +1,16 @@
-ARG APP_INSIGHTS_AGENT_VERSION=3.7.1
+FROM amazoncorretto:21-alpine
```

Comment:
Why have we added amazoncorretto and dockerfile changes?

Reply:
yeh, its this M4 needs the change, I'd forgotten and will remove this one, I'll sort out having 2 dockerfiles at some point.